src/views/client-dashboard/ClientPayments.vue

```diff
@@ -137,7 +172,27 @@
       }),
 
       getCardInfo(card) {
-        return `&#128179; ${card.brand && card.brand.toUpperCase()} **** ${card.last4} (exp: ${card.exp_month}/${card.exp_year})`;
+        const brand = ['amex', 'American Express'].includes(card.brand) ? 'AMEX' : card.brand.toUpperCase();
+
+        return `${brand} **** ${card.last4} (exp: ${card.exp_month}/${card.exp_year})`;
+      },
+
+      getImagePath(brand) {
+        const path = require.context('@/assets/svg');
+        let svg;
```

Comment:
How about:

`const localBrand = brand.toLowerCase();`

This way you will remove the 'Visa' and 'MasterCard' option.

Also, whenever you have a case, where 4 times you want to do something, 1 time something else, probably would make sense to just:

`const localBrand = brand.toLowerCase();`

`const path = ['visa', 'mastercard'].includes(localBrand) ? `./${localBrand}.svg` : './amex.svg'

Reply:
great